third_party/terraform/tests/resource_iam_workload_identity_pool_test.go

```diff
@@ -0,0 +1,55 @@
+package google
```

Comment:
You'll want to make this a `.go.erb` file and "beta guard" it so that the test doesn't appear at GA. See https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/tests/data_source_cloud_identity_groups_test.go.erb for an example.